libsolutil/InvertibleMap.h

```diff
@@ -18,77 +18,49 @@
 
 #pragma once
 
-#include <map>
-#include <set>
+#include <unordered_map>
 
 /**
  * Data structure that keeps track of values and keys of a mapping.
  */
 template <class K, class V>
 struct InvertibleMap
```

Comment:
Does the remaining code actually warrant its own class? The `fetch` function could be helpful in the global scope, maybe also providing a default value if the key is not present. `eraseValue` could also be helpful in the global scope, also if not that much.

Reply:
Yeah, it's mainly still here for easier rapid-changing - if we settle on any version of this, the class can be removed.
The ``fetch`` thing I'm actually not yet sure about - I have to remeasure performance on it and it only really works well on small cheaply-copiable value types... it would actually have to return ``std::optional<V const&>``, but unfortunately there's no such thing... we could go for a pointer, though...